roadmap/implementers-guide/src/node/utility/candidate-validation.md

```diff
@@ -12,12 +12,38 @@ Output: Validation result via the provided response side-channel.
 
 ## Functionality
 
-Given the hashes of a relay parent and a parachain candidate block, and either its PoV or the information with which to retrieve the PoV from the network, spawn a short-lived async job to determine whether the candidate is valid.
+This subsystem answers two types of requests. The goal of both request types is to validate a candidate. There are three possible outputs of validation: either the candidate is valid, the candidate is invalid, or an internal error occurred. Whatever the end result is, it will be returned on the response channel to the requestor.
```

Comment:
Explicitly state the both request types right there or at least in the same block.